File: Common/src/main/java/com/hypherionmc/sdlink/client/ClientEvents.java
package com.hypherionmc.sdlink.client;

import com.hypherionmc.sdlink.networking.SDLinkNetworking;

/**
 * @author HypherionSA
 * Client Side Functions (Mostly for mentioning Stuff)
 */
public class ClientEvents {
    public static boolean mentionsEnabled = false;

    public static void init() {
        SDLinkNetworking.registerPackets();
    }

}

File: Common/src/main/kotlin/com/hypherionmc/sdlinkrw/client/ClientEvents.kt
package com.hypherionmc.sdlinkrw.client

import com.hypherionmc.sdlink.networking.SDLinkNetworking

object ClientEvents {

    @JvmField
    var mentionsEnabled: Boolean = false

    fun init() {
        SDLinkNetworking.registerPackets()
    }

}